src/main/java/pl/arcadeit/forex/exception/UserExceptionHandler.java

```diff
@@ -1,17 +1,49 @@
 package pl.arcadeit.forex.exception;
 
+import org.springframework.context.support.DefaultMessageSourceResolvable;
 import org.springframework.http.HttpStatus;
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
 import pl.arcadeit.forex.controller.UserController;
 import pl.arcadeit.forex.model.ErrorMessage;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 @RestControllerAdvice(assignableTypes = UserController.class)
 public class UserExceptionHandler {
 
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ExceptionHandler(Exception.class)
+    public ErrorMessage handleGlobalException(final Exception exception) {
```

Comment:
proposal for the future: separate handleGlobalException to other class GlobalExceptionHandler